packages/applications-service-api/src/error/apiError.js

```diff
@@ -21,39 +25,27 @@ class ApiError {
 	}
 
 	static noApplicationsFound() {
-		return new ApiError(StatusCodes.NOT_FOUND, { errors: [`No applications found`] });
+		return this.notFound('No applications found');
 	}
 
 	static interestedPartyNotFound(caseRef) {
-		return new ApiError(StatusCodes.NOT_FOUND, {
-			errors: [`Interested party for project ${caseRef} not found`]
-		});
+		return this.notFound(`Interested party for project ${caseRef} not found`);
 	}
 
 	static noDocumentsFound() {
-		return new ApiError(404, { errors: [`No documents found`] });
-	}
-
-	static commentsForPartyWithIDNotUpdated(ID) {
```

Comment:
Is this a not found or could something else result in this error?

Reply:
I removed this `commentsForPartyWithIDNotUpdated` function as it doesn't seem to be in use anywhere